pallets/parachain-staking/src/migrations.rs

```diff
@@ -84,7 +86,8 @@ impl<T: Config> VersionMigratorTrait<T> for StakingStorageVersion {
 			Self::V1_0_0 => v2::pre_migrate::<T>(),
 			Self::V2_0_0 => v3::pre_migrate::<T>(),
 			Self::V3_0_0 => v4::pre_migrate::<T>(),
-			Self::V4 => Err("Already on latest version v4."),
+			Self::V4 => v5::pre_migrate::<T>(),
+			Self::V5 => Ok(()),
```

Comment:
Why are we not throwing here as we did before?

Reply:
test runtime upgrade will stop at the first failed migration. if we throw an error there we can't run any migrations. same for the other pallets